### PR TITLE
Update NFS

### PR DESCRIPTION
**Pull request by mmattel**
This updates the NFS prerequisites to be more precise.

Language review welcomed.

@dragonchaser fyi

### CODE REVIEW

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 1]
```diff
@@ -319,15 +321,32 @@ NFS Notes::
...
-Extended attributes are supported by NFS starting with kernel version 5.9, therefore the NFS server has to run on a system with kernel 5.9 or higher. To check, run the command `uname -a` on the NFS server and compare the displayed version number.
+NFS Storage Based on Linux Servers::
+Most easiest, when using a kernel version 5.9 or higher, extended attributes for NFS server and NFS client are part of the system. To check, run the command:
++
+[source,bash]
+----
+uname -r
+----
++
+On the system providing NFS server AND on the NFS client and check the displayed version number.
```
Comment (phil-davis) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:334]:
```suggestion
On the system providing the NFS server AND on the NFS client check the displayed version number.
```

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 2]
```diff
@@ -319,15 +321,32 @@ NFS Notes::
 +
 [NOTE]
 ====
-When using NFS, you have to take care that the NFS server provides `Extended Attributes`.
+When using NFS, you have to take care that the NFS server *AND* the NFS client provides `Extended Attributes`.
 
-NFS storage based on Linux servers::
-Extended attributes are supported by NFS starting with kernel version 5.9, therefore the NFS server has to run on a system with kernel 5.9 or higher. To check, run the command `uname -a` on the NFS server and compare the displayed version number.
+NFS Storage Based on Linux Servers::
+Most easiest, when using a kernel version 5.9 or higher, extended attributes for NFS server and NFS client are part of the system. To check, run the command:
```
Comment (phil-davis) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:327]:
```suggestion
When using a kernel version 5.9 or higher, extended attributes for NFS server and NFS client are part of the system. To check, run the command:
```

**Review by phil-davis — APPROVED** [review]

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 3]
```diff
@@ -319,15 +321,32 @@ NFS Notes::
...
+On the system providing the NFS server AND on the NFS client check the displayed version number.
+
+NFS Client::
+If you have an NFS server capable of extended attributes but you are unsure if your client accessing the server supports it, check the *nfs-utils* or *nfs-common* package version of your NFS client with the command:
++
+[source,bash]
+----
+mount.nfs -V
+----
++
```
Comment (dragonchaser) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:343]: Maybe an bash example how to check if extended attributes would be helpful, sth like:

```
$> touch foo.txt
$> attr -s foo -V bar foo.txt
Attribute "foo" set to a 3 byte value for foo.txt:
bar
$> attr -g foo foo.txt
Attribute "foo" had a 3 byte value for foo.txt:
bar
```

**Review by dragonchaser — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/prerequisites/prerequisites.adoc (hunk 3 above)
Comment (mmattel) [reply modules/ROOT/pages/prerequisites/prerequisites.adoc:372]: added, pls check

**Review by dragonchaser — APPROVED** [review]

**Pull request merged**